projects/plugins/jetpack/readme.txt

```diff
@@ -242,27 +242,71 @@ Jetpack Backup can do a full website migration to a new host, migrate theme file
 4. Promote your newest posts, pages, and products across your social media channels.
 
 == Changelog ==
-### 10.8-a.9 - 2022-03-23
+### 10.8-beta - 2022-03-29
 #### Major Enhancements
 - QR Post: add new feature which automatically generates QR codes for published posts. When scanned, the QR code will link visitors to the post. If a site has a custom logo set, it will be shown in the generated QR code image.
 
 #### Enhancements
+- Add back the upgrade nudge to the Payment Button block.
+- Adding language attributes to code blocks.
+- Dashboard: add a card to highlight the blocks available in Jetpack.
+- Dashboard: add highlight of the Jetpack blocks to the settings page.
+- Dashboard: add toggle to enable new Google Fonts feature.
+- Dashboard: show Search benefits on the Jetpack disconnection screen.
 - External Media: add Openverse as an external Media provider.
+- Form Block: offer option to install/activate the Jetpack CRM plugin from the Form Block options.
+- Jetpack: add QRPost feature which generates QR codes for published posts. Currently a JETPACK_BETA_BLOCKS feature.
+- Jetpack: assistant style updates and other improvements.
```

Comment:
I would leave this in "Other", since that's not really interesting for the end-user.